src/app/features/settings/settings-page/settings-form-component/settings-generic-component.css

```
@@ -0,0 +1,57 @@
+}
+
+.rowsContainer {
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+}
+
+.rowForm {
+  display: grid;
+
+  grid-template-columns: repeat(auto-fit, minmax(220px, 1fr));
```

Stylelint (17.6.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)